sql-db/sql-app/src/test/java/io/quarkus/ts/sqldb/sqlapp/DevModeMariadbDevServicesUserExperienceIT.java

```diff
@@ -37,9 +38,15 @@ public void verifyIfUserIsInformedAboutMariadbDevServicePulling() {
 
     @Test
     public void verifyMysqlImage() {
-        Image postgresImg = DockerUtils.getImage(MARIA_DB_NAME, MARIA_DB_VERSION);
-        Assertions.assertFalse(postgresImg.getId().isEmpty(), String.format("%s:%s not found. " +
+        Image image = DockerUtils.getImage(MARIA_DB_NAME, MARIA_DB_VERSION);
+        Assertions.assertFalse(image.getId().isEmpty(), String.format("%s:%s not found. " +
                 "Notice that user set his own custom image by 'quarkus.datasource.devservices.image-name' property",
                 MARIA_DB_NAME, MARIA_DB_VERSION));
     }
+
+    @AfterAll
+    //TODO workaround for podman 4.4.1 on rhel. Without it, *next* test (eg MariaDBDatabaseIT) fails with "broken pipe"
```

Comment:
This also happens with podman 4.5.0 (rootless).

Reply:
Hmm, I did not catch it neither on Fedora + 4.5.0 nor RHEL 9 + 4.6.0.dev. Could you share your environment and logs?

Reply:
Now these tests fail on 4.5.1, and were disabled completely